### PR TITLE
Difficulty/Duration

### PR DESCRIPTION
**Pull request by justinxreese**
For #161

### CODE REVIEW

**Comment by timothyfcook** [comment]
@justinxreese looks like a few conflicts to resolve then we can close #161

**Comment by justinxreese** [comment]
Cleaned it up :smile:

**Comment by justinxreese** [comment]
Oh darn, I dislike how Github handles this. Merged a new branch into master. That branch fixes #165

**Comment by MatthewVita** [comment]
@justinxreese hey, I don't see the merge in master?

**Comment by justinxreese** [comment]
No. You shouldn't. I was referring to on the builder fork. I don't like
merging my own PRs in to master. On you or someone else.

**Comment by chadwhitacre** [comment]
> I don't like merging my own PRs in to master.

Is there a reason someone else couldn't merge it?

**Comment by justinxreese** [comment]
@whit537 nope. Just feel like it's a system of checks and balances to not merge your own is all

**Comment by chadwhitacre** [comment]
Ah. I agree! :-)

File: app/views/opportunity_instances/_form.html.erb [hunk 1]
```diff
@@ -24,6 +24,14 @@
     <%= f.text_area :description %>
   </div>
   <div class="field">
+    <%= f.label :difficulty %><br>
+    <%= f.number_field :difficulty %>
+  </div>
+    <div class="field">
```
Comment (chadwhitacre) [inline app/views/opportunity_instances/_form.html.erb]: Indent level doesn't match.

File: app/views/opportunity_instances/_form.html.erb [hunk 2]
```diff
@@ -24,6 +24,14 @@
...
+    <%= f.label :duration %><br>
+    <%= f.number_field :duration %>
+  </div>
+    <div class="field">
```
Comment (chadwhitacre) [inline app/views/opportunity_instances/_form.html.erb]: Indent level doesn't match.

**Comment by justinxreese** [comment]
@whit537 fixed those indents

**Comment by chadwhitacre** [comment]
I'm happy to fully review/merge this on Friday if @timothyfcook would like. I'll get to learn some more Rails. :-)

**Comment by timothyfcook** [comment]
Sounds fine to me!

**Comment by timothyfcook** [comment]
Cool. Looks like we need to merge this. @whit537 can you go ahead? if not @MatthewVita can maybe get to it so we can close out #165 etc.

**Pull request merged**